specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/examples/AssessmentsMetadata/ListAssessmentsMetadata_example.json

```diff
@@ -0,0 +1,46 @@
+{
+  "parameters": {
+    "api-version": "2019-01-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Security/assessmentMetadata/21300918-b2e3-0346-785f-c77ff57d243b",
+            "name": "21300918-b2e3-0346-785f-c77ff57d243b",
+            "type": "Microsoft.Security/assessmentMetadata",
+            "properties": {
+              "displayName": "Install endpoint protection solution on virtual machine scale sets",
```

Comment:
same as the other file for CustomerManaged assessment

Reply:
not sure what you mean - I'll add the CustomerManaged example to the list